jenkins_pipelines/environments/manager-5.0-qe-build-validation-NUE

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env groovy
+
+node('sumaform-cucumber') {
+    properties([
+        buildDiscarder(logRotator(numToKeepStr: '10', daysToKeepStr: '30', artifactNumToKeepStr: '10')),
+        disableConcurrentBuilds(),
+        pipelineTriggers([cron('H H/4 * * *')]),
```

Comment:
Are we sure we want to run this pipeline using a cron ?

Reply:
Fixed.